splunk_add_on_ucc_framework/ucc_ui_lib/src/main/webapp/components/table/CustomTable.jsx

```diff
@@ -70,15 +124,36 @@ function CustomTable({ isInput, serviceName, data, handleToggleActionClick }) {
         );
     };
 
-    const handleEditActionClick = () => {};
+    const handleEditActionClick = (row) => { 
+        const temEntity ={...entityModal}
+        temEntity.open = true;
+        temEntity.serviceName = row.serviceName;
+        temEntity.stanzaName = row.name;
+        temEntity.mode = MODE_EDIT;
+        setEntityModal(temEntity);
 
-    const handleCloneActionClick = () => {};
+    };
 
-    const handleDeleteActionClick = () => {};
+    const handleCloneActionClick = (row) => {
+        const temEntity = {...entityModal}
+        temEntity.open = true;
+        temEntity.serviceName = row.serviceName;
+        temEntity.stanzaName = row.name;
+        temEntity.mode = MODE_CLONE;
+        setEntityModal(temEntity);
+    };
+
+    const handleDeleteActionClick = (row) => {
+        const temDelete = {...deleteModal}
```

Comment:
Can we also refactor this as the previous comment?

Reply:
done